crates/noirc_frontend/src/ast/statement.rs

```diff
@@ -238,6 +238,67 @@ pub enum PathKind {
     Plain,
 }
 
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub enum UseTree {
+    Simple(Path, Option<Ident>),
+    List(Path, Vec<PathListItem>),
```

Comment:
Can we make this recursive to allow for `use foo::{ bar::{baz, qux}, quux };`?

Reply:
Yes, something like this:

```rust
#[derive(Debug, PartialEq, Eq, Clone)]
pub struct UseTree {
    prefix: Path,
    kind: UseTreeKind,
}

#[derive(Debug, PartialEq, Eq, Clone)]
pub enum UseTreeKind {
    Simple(Option<Ident>),
    List(Vec<UseTree>),
}
```

I'll take care of it.